core/src/core/transaction.rs

```diff
@@ -453,19 +489,93 @@ impl ops::Add for SumCommit {
 	}
 }
 
-fn u64_to_32bytes(n: u64) -> [u8; 32] {
+/// TODO - consider putting this in consensus.rs so we can reuse it
+/// consistently across Transaction and TxKernel
+/// TODO - add some test coverage for this to make sure the bytes are in there
+/// correctly
+fn transaction_fee_and_lock_height_to_bytes(fee: u64, lock_height: u64) -> [u8; 32] {
```

Comment:
I'd use a name more descriptive of what's produced rather than consumed. Maybe `kernel_sig_msg`?

Reply:
yeah good catch - that's on my todo list actually (named it that so I knew what was going on...)